spec/lib/services/static-files-api-service-spec.js

```diff
@@ -0,0 +1,213 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+"use strict";
+
+describe("Http.Services.Api.StaticFiles", function () {
+    var path;
+    var staticFilesApiService;
+    var waterline;
+
+    before("Http.Services.Api.StaticFiles before", function() {
+        helper.setupInjector([
+            helper.require("/lib/services/static-files-api-service.js")
+        ]);
+        staticFilesApiService = helper.injector.get("Http.Services.Api.StaticFiles");
+        path = helper.injector.get('path');
+        waterline = helper.injector.get('Services.Waterline');
+        waterline.skus = {
+            findOne:function() {}
+        };
+        this.sandbox = sinon.sandbox.create();
+    });
+
+    afterEach("Http.Services.Api.StaticFiles afterEach", function() {
+        this.sandbox.restore();
+    });
+
+
+    describe("pairSkupackIds", function() {
+        var nodeFs;
+        before(function() {
+            nodeFs = helper.injector.get('fs');
+            sinon.stub(nodeFs, 'lstatAsync');
+            sinon.stub(nodeFs, 'readdirAsync');
+            this.sandbox.stub(waterline.skus, 'findOne');
+        });
+
+        beforeEach(function() {
+            nodeFs.lstatAsync.reset();
+            nodeFs.readdirAsync.reset();
+        });
+
+        after(function () {
+            nodeFs.lstatAsync.restore();
+            nodeFs.readdirAsync.restore();
+        });
+
+
+        it('should return an id:sku pair given valid directories and database', function() {
+            nodeFs.readdirAsync.resolves(['567432']);
+            nodeFs.readdirAsync().then(function (dirOutput) {
+                expect(dirOutput).to.deep.equal(['567432']);
```

Comment:
This assertion is not necessary here, it is only to test the line #50, but that line can be trusted.